packages/big-design/src/utils/messagingHelpers.tsx

```diff
@@ -25,6 +27,11 @@ export type MessageLinkItem = Pick<LinkProps, 'external' | 'href' | 'target'> &
   text: string;
 };
 
+export interface MessageAction extends Omit<ButtonProps, 'children' | 'mobileWidth' | 'variant'> {
+  text: string;
+  variant?: Extract<ButtonProps['variant'], 'secondary' | 'subtle'>;
```

Comment:
🍹 Maybe use `Exclude<ButtonProps['variant'], 'primary'>` in case we add more variants to buttons in the future?

Reply:
I was thought that we should support only `secondary` and `subtle` variants 😄  So, yeah, it's not a problem to change it 🤝